FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderTests.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import FirebaseCore
+@testable import FirebaseMLModelDownloader
+
+final class ModelDownloaderTests: XCTestCase {
+  /// Unit test for reading and writing to user defaults.
+  func testUserDefaults() {
+    FirebaseApp.configure()
+    let testApp = FirebaseApp.app()!
+    let testModelName = "user-defaults-test-model"
+    let modelInfoRetriever = ModelInfoRetriever(
+      app: testApp,
+      modelName: testModelName,
+      defaults: .getTestInstance()
+    )
+    modelInfoRetriever.modelInfo = ModelInfo(app: testApp, name: testModelName)
+    // XCTAssertEqual(modelInfoRetriever.modelInfo?.downloadURL, "")
```

Comment:
I figured this line would work, given that `getTestInstance()` wipes out user defaults, but it fails. What am I missing?